articles/openshift/oshift-scaling-references.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Considerations When Scaling an OpenShift Cluster | UKCloud Ltd
+description: Gives information on things to consider when scaling a cluster
+services: openshift
+author: Kieran O'Neill
+toc_rootlink: References
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Considerations When Scaling an OpenShift Cluster
+toc_fullpath: References/oshift-scaling-references.md
+toc_mdlink: oshift-scaling-references.md
+---
+
+# Considerations When Scaling an OpenShift Cluster
+
+### Infrastructure applications
+
+With your UKCloud for OpenShift cluster we deploy a number of infrastructure applications by default. These are quality of life features that are intended to help you with the administration, management and application development within your cluster. Examples are the EFK stack, Hawkular Metrics, the integrated registry, the service catalog and; with v3.11 deployments onwards, Prometheus and Grafana.
+
+These applications run their components on the infrastructure nodes (these are the nodes with the naming convention worker-infra) your cluster comes out of the box with 2 of these nodes and each has 16GB of memory and 2 vCPUs. This is a perfectly reasonable amount of resource for the infrastructure components for a small cluster running light development workloads. If you are looking to scale out the cluster to a larger size and start running more intensive applications or builds these nodes may not be enough. For example, the more applications and nodes you have in your environment the more logs will be gathered off the nodes by fluentd and sent off to Elasticsearch for indexing. This will result in significantly higher resource usage. In the same vein, the more pods and nodes are running in your environment the more stats hawkular metrics will be gathering and storing in the Cassandra database.
```

Comment:
I think you need some extra commas in the last few sentences here:

If you're looking to scale out the cluster to a larger size and start running more intensive applications or builds, these nodes may not be enough. For example, the more applications and nodes you have in your environment, the more logs will be gathered off the nodes by fluentd and sent off to Elasticsearch for indexing. This will result in significantly higher resource usage. In the same vein, the more pods and nodes are running in your environment, the more stats hawkular metrics will be gathering and storing in the Cassandra database.